CONTRIBUTING.md

```diff
@@ -38,9 +38,16 @@ This repo is a “monorepo,” meaning it contains several projects in one. It c
 
 ### Setting up a development environment
 
+You can [develop locally](#developing-locally) or use an online coding development environment like [GitHub Codespaces](#developing-using-github-codespaces) or [Gitpod](#developing-using-gitpod) to get started quickly.
+
 #### Developing locally
 
-**Prerequisites:** Developing Starlight requires [Node.js](https://nodejs.org/en) and [pnpm](https://pnpm.io/). Make sure you have these installed before following these steps.
+**Prerequisites:** Make sure you have these installed before continuing.
+
+```shell
+node: "^>=16.12.0"
+pnpm: "^8.2.0"
+```
```

Comment:
It’s a shame to lose the links as those both include install instructions for those who need them. How about this if we want to mention the supported versions?

```suggestion
**Prerequisites:** Developing Starlight requires [Node.js](https://nodejs.org/en) (v16 or higher) and [pnpm](https://pnpm.io/) (v8.2 or higher). Make sure you have these installed before following these steps.
```

Reply:
I personally agree. Changed to be consistent with https://github.com/withastro/astro/blob/main/CONTRIBUTING.md#prerequisites .